config/purposes/scrna_core_library_prep.yml

```diff
@@ -116,8 +116,9 @@ LRC GEM-X 5p CITE SUP:
   :input_plate: false
 LRC GEM-X 5p CITE SUP XP:
   :asset_type: plate
-  :stock_plate: false
-  :input_plate: false
+  :stock_plate: true # Uploaded via sample manifest.
```

Comment:
LRC GEM-X 5p CITE SUP XP is not uploaded via sample manifest; It is child of LRC GEM-X 5p CITE SUP . The stock_plate flag should be false I think.